internal/pkg/service/stream/storage/repository/file/fixtures/file_retry_snapshot_002.txt

```diff
@@ -0,0 +1,23 @@
+<<<<<
+storage/file/level/target/123/456/my-source/my-sink/2000-01-01T01:00:00.000Z
+-----
+{
+  %A
+  "state": "imported",
+  "closingAt": "2000-01-01T02:00:00.000Z",
+  "importingAt": "2000-01-01T05:00:00.000Z",
+  "importedAt": "2000-01-01T08:00:00.000Z",
+  %A
+}
+>>>>>
+
+<<<<<
+storage/file/level/local/123/456/my-source/my-sink/2000-01-01T02:00:00.000Z
+-----
+{
+  %A
+  "state": "writing",
+  %A
+}
```

Comment:
Here are file states with description:
https://github.com/keboola/keboola-as-code/blob/331609a7b783daf22e2ba1f30b958a3bd6f2b2f9/internal/pkg/service/stream/storage/model/filestate.go#L12-L48

Reply:
Just a quick question. When we are importing the file, we do it slice by slice or as whole file? Just thinking about transtition of `FileImporting` -> `FileImported`. As the Slices are uploaded already it should be just a `state` change in case of `FileImporting` and `FileImported`.

Reply:
> When we are importing the file, we do it slice by slice or as whole file?

We say "Snowflake/BigQuery" here is S3/ABS/GCS prefix with all uploaded CSV slices, import them into the table in parallel.